### PR TITLE
index.html page added

### PR DESCRIPTION
**Pull request by Ndaboom**
Technical documentation navigator and content added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# technical-documentation
-A technical documentation template web app...
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (Ndaboom) [inline README.md:5]: * Kindly remove unnecessary details in your README.md and make it looks more professional. Check out this [repo](https://github.com/Ndaboom/portofolio) for an overview of what it's supposed to look like😋

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@media (min-width: 1281px) {
+#navbar{
+    width: 20%;
+    background-color: gray;
+    height: 100%;
+    background-color: white;
+}
+#main-doc{
+    width: 80%;
+}
```
Comment (Ndaboom) [inline css/style.css:10]: * Please use a more semantic way of writing CSS. Here I'm talking about indentation between your CSS selectors and braces 👌🏾🤗. Check out this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+nav-link {
+    height: 30px;
+    line-height: 30px;
+}
```
Comment (Ndaboom) [inline css/style.css:15]: * Kindly notice that this selector has a . missing on its start, please adjust that 😎 .

**Review by Ndaboom — COMMENTED** [review]
Changes Required ♻️ : 

Hi @Ndaboom 👋, Good job so far!
Your project is looking fantastic so far 🤩



However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional 😎

* Please add linter, which is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Check out this [link ](https://github.com/microverseinc/linters-config) for the linter setup.

**Review by alexpuente0 — APPROVED** [review]
Hi, @Ndaboom!



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Your comments were polite and friendly
- You used the inline comments correctly

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**